Tests/PackageGraphTests/ModulesGraphTests+Traits.swift

```diff
@@ -0,0 +1,1053 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift open source project
+//
+// Copyright (c) 2025 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Basics
+import PackageModel
+import TSCUtility
+import Testing
+import _InternalTestSupport
+
+@_spi(DontAdoptOutsideOfSwiftPMExposedForBenchmarksAndTestsOnly)
+@testable import PackageGraph
+
+extension ModulesGraphTests {
```

Comment:
suggestion: consider create a new structure and adding the `.Feature.Traits` tag to it.